app.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const session = require('express-session');
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser')
+const { check, validationResult } = require('express-validator');
+
+const users = require('./routers/user.routers.js');
+const currentUsers = require('./routers/currentUser.routers.js');
+const products = require('./routers/product.routers.js');
+const shops = require('./routers/shop.routers.js');
+const senders = require('./routers/sender.routers.js');
+const sessions = require('./routers/sessions.routers.js');
+const env = process.env.NODE_ENV || 'development';
```

Comment:
did you set anywhere NODE_ENV value? i.g: did you set it to be 'production' for production build?